content/docs/manual/access-control.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Access Control"
+weight : 3900
+menu:
+  docs:
+    parent: manual
+---
+
+*NOTE: This documentation covers the Zenoh 1.0 ACL config. For Zenoh 0.11 ACL config,
+please refer to the [Zenoh 0.11 Access Control Rules RFC](https://github.com/eclipse-zenoh/roadmap/blob/ca841fe219890bf73289089b520271d70ded89b6/rfcs/ALL/Access%20Control%20Rules.md)*
+
+*Access control* enables Zenoh instances to filter (allow or deny) messages,
+depending on certain characteristics of individual messages and their respective source or destination.
+*Authentication* on the other hand allows Zenoh instances to identify certain characteristics in other instances they connect to,
+which are used to match the instances with configured subjects in the ACL policies and apply the rules accordingly on the exhanged messages.
```

Comment:
which are used to match the foreign instances or the local instance ?
My guess is that a Local instance is able to match foreign instances with configured subjects, but maybe be explicit because on line 14 you've used `instances` twice, so im not sure which group you are referring to.

Reply:
Indeed it's confusing. I'll change it to say `remote instances` in line 15.